web/src/routes/(user)/albums/+page.svelte

```diff
@@ -1,11 +1,20 @@
+<script lang="ts" context="module">
+  // table is the text printed in the table and order is the text printed in the dropDow menu
+  export interface Sort {
+    table: string;
+    dropDow: string;
```

Comment:
spelling, should be `dropDown` probably. But `label` or `title` might be more appropriate. What about `displayTitle` and `sortTitle`? Maybe also `widthClass`

Reply:
This is much better. Thanks :pray: